src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsuofdataapi/utils/UserContextFilter.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsuofdataapi.utils
```

Comment:
Not sure we need these as we are not calling another API

Reply:
Removed